src/application/utils/transaction.ts

```diff
@@ -140,7 +83,7 @@ export const fillTaxiTx = (
 
```

Comment:
Please add types to fillTaxiTx params.
coinSelector: CoinSelector,
changeAddressGetter: ChangeAddressFromAssetGetter